app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -70,19 +81,31 @@
                     <div id="<%= "lookupModal#{modal_index}" %>" class="modal fade" role="dialog">
                       <%= render partial: 'checklist_modal', locals: { valuessets: valuessets, product_test: product_test, index: modal_index, is_attribute: false } %>
                     </div> 
+                    <% end %>
                   </td>
                   <td>
                     <% if criteria_field.object.complete?.nil? %>
                     <i class = 'fa fa-fw fa-play-circle text-info invisible' aria-hidden="true"></i>
+                    <% elsif criteria_field.object.negated_valueset %>
+                    <i class = 'fa fa-fw fa-check text-success' aria-hidden="true"></i>
                     <% elsif criteria_field.object.code_complete %>
                     <i class = 'fa fa-fw fa-check text-success' aria-hidden="true"></i>
                     <% else %>
                     <i class = 'fa fa-fw fa-times text-danger' aria-hidden="true"></i>
                     <% end %>
-                    <%= criteria_field.text_field :code, hide_label: false %>
+                    <% if criteria_field.object.negated_valueset %> 
+                   <%= criteria_field.select :selected_negated_valueset, valuessets.collect{ |vs| ["#{lookup_valueset_name(vs)} - #{vs}", vs] } %>                    
+                    <% else %>
+                      <%= criteria_field.text_field :code, hide_label: false %>
+                    <% end %>
```

Comment:
Same indentation inconsistency here, regarding the if else block, as well as spacing between <% and the keyword (especially since in one spot, a change was made to simply cut the leading and trailing spaces.